src/components/Button/Button.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../styles/variables";
+@import "../../styles/mixins";
+
+.button {
+  width: 100px;
+  padding: 10px;
+  font-size: 16px;
+  background: transparent;
+  border: none;
+  position: relative;
+  color: $colorBtnFont;
+  z-index: 1;
+
+  &::after {
+    @include absolute-bottom-right-z;
+    -webkit-transform: translate(10px, 10px);
+    -ms-transform: translate(10px, 10px);
+    transform: translate(10px, 10px);
+    width: 35px;
+    height: 35px;
+    background: $colorBtnBlur;
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    border-radius: 50px;
+  }
+
+  &::before {
+    @include absolute-bottom-right-z;
+    @include before-settings;
```

Comment:
Mixins should be named with a verb or verb phrase and should be camelCased. The mixin 'before-settings' seems to be in kebab-case.